dotnet/test/AutoGen.Core.Tests/GroupChat/GraphTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace AutoGen.Core.Tests
+{
+    [TestClass()]
+    public class GraphTests
```

Comment:
Could you move this test to `AutoGen.Tests` project? That's where we usually put AutoGen.Core's tests

Reply:
OK，I will move it.